### ISSUE TITLE
Chunk border cuts through all unloaded chunks on a current world.

### ISSUE THREAD

**Issue by MithMax** [finallion/The-Graveyard-FORGE-#33]
### What's the issue you encountered?

Straight cuts through all unloaded chunks, and a lot of console spam starting at 16:34:38] 
The chunk line It is visible on the minimap as well in the picture.
![2022-03-19_16 46 24](https://user-images.githubusercontent.com/97056197/159138216-3a27b207-3514-454e-8291-8f13c1020426.png)

[latest.log](https://github.com/finallion/The-Graveyard-FORGE-/files/8310125/latest.log)

### How can the issue be reproduced?

Loading 1.6 Graveyards (does not need the new graveyard biomes or terrablender to reproduce) in a current world. Not using any other terrain generating mods. No problems with 1.5 version.
The log will be showing biomes and terrablender loading, but I have reloaded a world and started with only Graveyard 1.6 and my other mods. with the same chunk error problems, and odd console spam.
I have been copying and pasting a new, current world folder each time I test as well.

I also removed all mods except Terrablender, Graveyard1.6,  Graveyard biomes, and Geckolib to test and then with just Graveyard 1.6, and Geckolib and still had the same results, just to eliminate a mod conflict.

### Logs

[latest.log](https://github.com/finallion/The-Graveyard-FORGE-/files/8310125/latest.log)

### Additional information

I am going to edit my post under graveyard biomes (forge version) on curseforge with like a happy face and nice work or something! As this issue is not related to the graveyard biomes anyway. It wont let me simply delete the post over there.

My family and I love this mod. I hope there is a way to fix this for current worlds. Thanks for your hard work on this.

**Comment by finallion** [comment]
Hey! It is actually an issue related to the Graveyard Biomes. Due to it being an own mod, the biome ID for each biome changed and therefor minecraft thinks that the "old" biomes from the Graveyard were removed and "new" ones from the Graveyard Biomes were added. The removal of biomes causes chunk borders.
I'm really sorry for this. It is an unfortunate and sadly not fixable issue due to the migration (which was for other reasons necessary) to an own mod. If you want to avoid chunk borders you should stay on 1.5 if it is possible for you.

I'll will make the notification on the curseforge page to make backups before updating clearer.

**Comment by MithMax** [comment]
So the only real  issue were the player built structures and items. So, I just learned how to world edit. I had everyone login to save the items on their characters in a chest. I Ran a region editor Saved all the built structures, and  items within the selected regions. Loaded a new world with our current seed. Saved, then shutdown.  Then moved my edited regions into the new world save region folder. So everyone will have to get new horses and pets, but the structures and items are all there within the saved regions and now 1.6 with the biomes is running perfectly now. So don't be sorry.. I appreciate the quick response, and I learned some things in the process! I appreciate the work your doing, Thank you.
